pkgs/applications/audio/samplebrain/default.nix

```
@@ -0,0 +1,56 @@
+{
+  fetchFromGitLab,
+  fftw,
+  lib,
+  liblo,
+  libsndfile,
+  makeDesktopItem,
+  mkDerivation,
+  portaudio,
+  qmake,
+  qtbase,
+  wrapQtAppsHook,
+}:
+mkDerivation rec {
+  pname = "samplebrain";
+  version = "0.18";
+  src = fetchFromGitLab {
+    owner = "then-try-this";
+    repo = "samplebrain";
+    rev = "${version}_release";
+    sha256 = "sha256-k5VFaIo6XKv9n6OQNNDLVneKQDBZA4eFu0RNea9DpVM=";
+  };
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    fftw
+    liblo
+    libsndfile
+    portaudio
+    qtbase
+  ];
+  desktopItem = makeDesktopItem {
+    type = "Application";
+    desktopName = "${pname}";
+    name = "${pname}";
+    comment = "A sample masher designed by Aphex Twin";
+    exec = "${pname}";
+    icon = "${pname}";
+    categories = [ "Audio" ];
+  };
+  installPhase = ''
+    mkdir -p $out/bin
+    cp samplebrain $out/bin
+    install -m 444 -D ${desktopItem}/share/applications/${pname}.desktop $out/share/applications/${pname}.desktop
+    install -m 444 -D desktop/${pname}.svg $out/share/icons/hicolor/scalable/apps/${pname}.svg
+  '';
```

```suggestion
  };

  installPhase = ''
    mkdir -p $out/bin
    cp samplebrain $out/bin
    install -m 444 -D desktop/samplebrain.svg $out/share/icons/hicolor/scalable/apps/samplebrain.svg
  '';
  
```
Please use copyDesktopItems to copy the generate desktop file(s)